db/seeds/playbooks/flake.seeds.rb

```diff
@@ -0,0 +1,256 @@
+# frozen_string_literal: true
+
+after :playbook do
+  @flake = Playbook.find_by! name: 'The Flake'
+
+  #####
+  # Moves
+  #####
+  [
+    {
+      name: 'Connect the Dots',
+      type: 'Moves::Rollable',
+      rating: :sharp,
+      description: ' At the beginning of each mystery,
+      if you look for the wider patterns that current
+      events might be part of. Spend your hold during the
+      mystery to ask the Keeper any one of the following
+      questions:
+      - Is this person connected to current events more than they are saying?
+      - When and where will the next critical event occur?
+      - What does the monster want from this person?
+      - Is this connected to previous mysteries we have investigated?
+      - How does this mystery connect to the bigger picture?',
+      seven_to_nine: 'On a 7-9 hold 1',
+      ten_plus: 'On a 10+ hold 3'
+    },
+    {
+      name: 'Crazy Eyes',
+      type: 'Moves::Descriptive',
+      description: 'You get +1 Weird (max +3).'
+    },
+    {
+      name: 'See, It All Fits Together',
+      type: 'Moves::Descriptive',
+      description: 'You can use Sharp instead of Charm when you manipulate someone.'
+    },
+    {
+      name: 'Suspicious Mind',
+      type: 'Moves::Descriptive',
+      description: 'If someone lies to you, you know it.'
+    },
+    {
+      name: 'Often Overlooked',
+      type: 'Moves::Rollable',
+      rating: :weird,
+      description: 'When you act all crazy to avoid something.',
```

Comment:
Let's include the 'roll + Weird' here, it'll look really strange on the moves section without it.